### PR TITLE
Update dependency versions

### PR DESCRIPTION
**Pull request by lwgordonimos**
Don't merge yet, needs to be tested alongside aodndata/aodnfetcher

### CODE REVIEW

**Comment by lwgordonimos** [comment]
Good question! I think we'd kind of touched on it before, but we really need to test aodndata against the build and systest artifacts as well, so we can get travis to pick up test failures *before* promoting this to prod.

So I think before proceeding with this one, we should have a look at the travis.yml in aodndata to get it testing against aodncore_build, aodncore_systest and aodncore_prod artifacts to give us a heads up on any breaking changes.

File: setup.py [hunk 1]
```diff
@@ -10,20 +10,20 @@
...
     ':python_version < "3.3"': ['mock == 2.0.0'],
     ':python_version < "3.4"': ['enum34==1.1.6'],
-    ':python_version < "3.5"': ['scandir == 1.6', 'typing==3.6.4'],
+    ':python_version < "3.5"': ['scandir == 1.9.0', 'typing==3.6.6'],
 }
 
 INSTALL_REQUIRES = [
-    'boto3==1.4.4',
-    'celery==4.1.1',
+    'boto3==1.9.27',
```
Comment (mhidas) [inline setup.py]: This needs to be compatible with `boto3>=1.9.156` requirement of aodnfetcher.

**Comment by mhidas** [comment]
> So I think before proceeding with this one, we should have a look at the travis.yml in aodndata to get it testing against aodncore_build, aodncore_systest and aodncore_prod artifacts

This is now done. Can we update this PR and test it out?

**Comment by mhidas** [comment]
I've merged  #156 which updated the boto3 version, so this PR needs rebasing. 
@lwgordonimos Is it still worth pushing this one through?

**Comment by lwgordonimos** [comment]
Closing as we will revisit this shortly with Python 3 work.

**Pull request closed without merging**